Technical-Steering-Committee/logging.md

```diff
@@ -0,0 +1,46 @@
+# Message Management
+
+## How do we log
+
+### Message ID Components (message prefixes)
+
+OMP - Open Mainframe Project
+ZWE - Zowe, This should be used as default for all Zowe messages. 
+
+### Zowe server side Projects
```

Comment:
I think this heading is confusing. Only the API Mediation Layer, App Framework, and ZSS are server side projects.

Reply:
Thanks, I missed this from previous iteration.